index.html

```diff
@@ -0,0 +1,180 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="import" href="https://googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Your-Tube-MV</title>
+</head>
+<body>
+    <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+    <header class="header">
+        <div class="logo left">
+            <i id="menu" class="material-icons">menu</i>
+            <img src="https://www.freecodecamp.org/news/content/images/2022/01/yt-logo.png" alt="youtube logo">
+        </div>
+        <div class="search center">
+            <form action="">
+                <input type="text" placeholder="Search"/>
+                <button><i class="material-icons">search</i></button>
+            </form>
+            <i class="material-icons mic">mic</i>
+        </div>
+        <div class="icons-right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons display-this">account_circle</i>
+        </div>
+       
+
+    </header>
+   
+    <main>
+        
+        <div class="sidebar" class="nav">
+            <a class="nav_link active">
+                <i class="material-icons">home</i>
+                <span>Home</span>
+            </a>
+            <a class="nav_link">
+                <i class="material-icons">local_fire_department</i>
+                <span>Trending</span>
+            </a>
+            <a class="nav_link">
+                <i class="material-icons">subscriptions</i>
+                <span>Subscriptions</span>
+            </a>
+        </div>
+        <hr>
+        <a class="nav_link">
+            <i class="material-icons">library_add_check</i>
+            <span>Library</span>
+        </a>
+        <a class="nav_link">
+            <i class="material-icons">history</i>
+            <span>history</span>
+        </a>
+        <a class="nav_link">
+            <i class="material-icons">play_arrow</i>
+            <span>Your videos</span>
+        </a>
+        <a class="nav_link">
+            <i class="material-icons">watch_later</i>
+            <span>Watch Later</span>
+        </a>
+        <a class="nav_link">
+            <i class="material-icons">thumb_up</i>
+            <span>Liked</span>
+        </a>
+        <div class="content">
+            <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+            <div class="details">
+                <div class="Author">
+                    <img src="https://yt3.ggpht.com/bpzY-S4DYlbTeOpY5hIA7qz_hcbMkgvLAugtwKBGTTImNnWAGudX0y53bo_fJZ0auypxrWkUiw=s88-c-k-c0x00ffffff-no-rj" alt="">
+                </div>
```

Comment:
there is a need to use good/ proper indentation as per good practices of HTML and CSS. It is good effort but I suggest that you check the indentation and you may change it.

Reply:
- [ ] kindly. consider making your comments clear and easy to understand cause currently here you are confusing the student .where should him/her check for indentation also remember its always good  to give a hint on how it can be solved 
- example 👇 
```
**This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

**This is a vague comment: The README is not descriptive, why don't you make it better?
```